airflow/models.py

```diff
@@ -3921,7 +3921,7 @@ class XCom(Base):
 
     id = Column(Integer, primary_key=True)
     key = Column(String(512))
-    value = Column(PickleType(pickler=dill))
+    value = Column(LargeBinary)
```

Comment:
Ok, we should make sure the DB migration does something reasonable. (ie not deleting values)